components/automate-chef-io/content/docs/configuration.md

```diff
@@ -243,16 +243,18 @@ policies will be incorrectly labelled as `ldap`.
 
 Chef Automate comes with a default LDAP configuration for Microsoft Active Directory (MSAD).
 The MSAD configuration is intended for getting started with minimal configuration for standard MSAD systems.
-If you need to customize your configuration further, see the [LDAP section]({{< relref "configuration.md#LDAP" >}}) below.
+The MSAD configuration makes some assumptions about your MSAD setup, but you can override those defaults.
 
 {{< warning >}}
 Chef Automate's default configuration for Microsoft AD is specific to LDAP.
 To configure Microsoft AD using SAML, see the SAML configuration section below.
```

Comment:
Please include an internal link to the SAML configuration section for ease of reference.
```suggestion
To configure Microsoft AD using SAML, see the SAML configuration section.
```